tasks/pe_uninstall.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Uninstalling PE with the following flags
+# For more information about the uninstaller
+#  and the command-line flags, visit:
+#  https://puppet.com/docs/pe/2019.8/uninstalling.html
+
+if [ -x "$(command -v /opt/puppetlabs/bin/puppet-enterprise-uninstaller)" ]; then
+    sudo /opt/puppetlabs/bin/puppet-enterprise-uninstaller -d -p -y
```

Comment:
Tasks should not use `sudo`. Bolt will handle privilege escalation itself if necessary.

```suggestion
    /opt/puppetlabs/bin/puppet-enterprise-uninstaller -d -p -y
```

Reply:
Good catch @reidmv !